evaluation.md

```diff
@@ -6,8 +6,59 @@ _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
 **On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
+10. The syntax is not as clear as we wanted it to be initially. Now we're relying
+on one method, `If`, that calls every other method.  Thus, we're passing in strings as
+parameters and calling the other relevant functions, `Nothing`, `Something`, and `Go` from
+the method `If`. 
+
+We've changed from having 
+```
+North = {
+    if (Nothing[N]) Go North
+    if (Something[N] and Nothing [W]) Go West
+    if (Something[N, W]) Go East
+}
+``` 
+to represent all the rules for going North, to having 
+```
+If ("N", "Nothing(N)","Go(North)")
+If ("N", "Something(N) and Nothing(W)", "Go(West)")
+If ("N", "Something(N, W)", "Go(East)")
+```
+Now, strings have to be passed in to the main function `If`. The readability is
+definitely hurt since we moved the `North` header into a parameter for `If`. 
```

Comment:
We had a similar idea where we wanted to reduce the amount of redundancy be allowing users to type in a set of rules corresponding to a state, very similar to what you did in the ideal syntax. But we also ended up requiring users to type in the state for every rule.